src/static/js/modules/jquery/cf_notifier.js

```diff
@@ -7,8 +7,11 @@
 var $ = require( 'jquery' );
 var handlebars = require( 'handlebars' );
 
-var _notifierTemplate = '<div class="cf-notification cf-notification__{{ state }}" style="display: none;">' +
-  '<span class="cf-notification_icon cf-notification_icon__{{ state }} cf-icon cf-icon-{{ icon }}-round"></span>' +
+var _notifierTemplate = '<div class="cf-notification cf-notification__{{ state }}" ' +
+                             'style="display: none;">' +
+  '<span class="cf-notification_icon ' +
```

Comment:
I kinda matched how we indent our markup. Wasn't sure how to do this one really, so open to suggested examples.